pkg/api/kubegraph/analysis/hpa_test.go

```diff
@@ -0,0 +1,28 @@
+package analysis
+
+import (
+	"testing"
+
+	osgraph "github.com/openshift/origin/pkg/api/graph"
+	osgraphtest "github.com/openshift/origin/pkg/api/graph/test"
+)
+
+func TestHPAMissingCPUTargetWarning(t *testing.T) {
+	g, _, err := osgraphtest.BuildGraph("./../../../api/graph/test/hpa-missing-cpu-target.yaml")
```

Comment:
I don't think it's possible in the API, but it will be as soon as we get the autoscaling group in the rebase :-/